tiny/config.yml

```diff
@@ -39,6 +39,9 @@ defaults:
 # Where to put log files
 log_dir: '{}'
 
+# (Optional) Limits the maximum number of messages stored per each UI tab
+# max_buffer_per_tab: 512
```

Comment:
Could leave this uncommented and just blank. I believe it makes it None

Reply:
In the example config, we should note that the default is unlimited.

Reply:
Let's call this "scrollback" or "history". The word "scrollback" is used in konsole, hexchat, konversation and probably many other programs, and in konversation and hexchat they don't even explain what it is about because it's so obvious. "history" is used by e.g. tmux.

Personally I prefer "scrollback" as that's the word I use to refer to this concept all the time, so it's familiar to me.

Also a nit: s/per each/for each